build/yamls/windows/ovs-containerd/conf/Install-WindowsCNI-Containerd.ps1

```diff
@@ -0,0 +1,13 @@
+$ErrorActionPreference = "Stop";
+mkdir -force c:/var/log/antrea
```

Comment:
It is not necessary to copy Install-WindowsCNI-Containerd.ps1 and Run-AntreaAgent-Containerd.ps1 to this path, you can involve them in the kustomization.yml like how the files in `base` are referrenced?

Reply:
Do you research if customization could support merging the files in different directory to one automatically?

Reply:
done, PTAL.